### PR TITLE
144 - Get list of all connectors

### LINKED ISSUE #144: ✨ Get list of all connectors
> **As a** User
> **I want** see a list of all connectors and their settings
> **so that** I can see what is going on

## Description:
With this user story the discord "tab" will be replaced with the new connectors "tab". The user will see a redesigned "Discord" Panel, which is modular so it can be easily used for other types of connectors as well.

grafik

### 🟢 In scope:
* Implement `GET /api/connectors`
* Prepare design (buttons and switches) for next user stories
* Hide secrets like the discord token

### 🔴 Not in scope:
* Do not implement other endpoints
* Update Documentation
* Handle courses array

### PR DESCRIPTION
**Pull request by tjarbo**
> **As a** User
> **I want** see a list of all connectors and their settings
> **so that** I can see what is going on

## Description
Closes #144
Know the user can see a list of all registered connectors on the dashboard. As the user story describes, it's just the "R" of "CRUD" for now. A new endpoint `GET /api/connectors` has been implemented. The old discord endpoints will be replaced with #145

### Screenshots
grafik

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Checklist

- [ ] ~Make sure you are requesting to main. Don't request to release branch!~
- [X] I have linked the related issue.
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/backend/missing.d.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// See https://github.com/alonronin/mockingoose/issues/70
+declare module 'mockingoose';
```
Comment (antonplagemann) [inline packages/backend/missing.d.ts:2]:
```suggestion
declare module 'mockingoose';

```

**Review by antonplagemann — APPROVED** [review]
👍

**Comment by tjarbo** [comment]
Unit Tests in their current configuration do not bring that much benefit, so I decided to skip them. Maybe we should apply jest as the unit-testing framework for the front-end as well to keep the consistency.

**Review by antonplagemann — APPROVED** [review]
👍

**Review by piuswalter — APPROVED** [review]

**Pull request merged**